README.md

```diff
@@ -61,231 +30,178 @@ For more information on the importance of a professional README for your reposit
 - [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
-<!-- PROJECT DESCRIPTION -->
 
-# 📖 [your_project_name] <a name="about-project"></a>
+# 📖 [Capstone-Project] <a name="about-project"></a>
 
-> Describe your project in 1 or 2 sentences.
 
-**[your_project__name]** is a...
+**[Capstone-Project]** This is a collaborative project where we have applied all the skills obtained throughout the javascript module.
```

Comment:
- [ ] Give a proper name to the project "Capstone-Project" is not sound good,& give a proper description of the project not about collaboration.